schema.sql

```diff
@@ -48,4 +48,11 @@ CREATE TABLE visits(
   animals_id INTEGER REFERENCES animals (id),
   vets_id INTEGER REFERENCES vets(id),
   date_of_visit DATE
-);
\ No newline at end of file
+);
+
+--Week 2 day 1 --database performance audit--
+CREATE INDEX animals_id_idx on visits (animals_id);
+
+CREATE INDEX vets_id_idx ON visits (vets_id);
+
+CREATE INDEX idx_owners_email ON owners (email);
```

Comment:
- [ ] [OPTIONAL] Good job so far! As email addresses are not case-sensitive, we might need to look for the emails in a case-insensitive way. There is an option to reflect that in the index by using:

```sql
CREATE INDEX ON owners ((lower(email)));
```